### PR TITLE
Add error messages

### PR DESCRIPTION
**Pull request by NickColley**
Fleshes out some of the error validation, ensures that the error summary links go to the correct fields.

Will need another pass at this once the questions have been finalised.

### CODE REVIEW

File: app/controllers/coronavirus_form/support_address_controller.rb [hunk 1]
```diff
@@ -52,17 +52,15 @@ def validate_conditionally_present_fields(product)
...
-      }
-    end
-
     if product["county"].blank?
       invalid_fields << {
         field: "county",
-        text: t("coronavirus_form.errors.missing_county_field"),
+        text: t("coronavirus_form.errors.missing_county_or_postcode_field"),
+      }
+    elsif product["postcode"].blank?
```
Comment (NickColley) [inline app/controllers/coronavirus_form/support_address_controller.rb:60]: Only show one error in the error summary

File: app/views/coronavirus_form/support_address.html.erb [hunk 2]
```diff
@@ -22,19 +22,22 @@
 <%= form_tag({},
   "data-module": "track-coronavirus-form-vulernable-people-support-address",
   "data-question-key": "support_address",
-  "id": "support_address"
+  "id": "support_address",
+  "novalidate": "true"
```
Comment (injms) [inline app/views/coronavirus_form/support_address.html.erb:26]: I did not know you could tell the browser not to validate - thanks!

File: config/locales/en.yml [hunk 3]
```diff
@@ -58,7 +53,7 @@ en:
             label: "No"
           not_sure:
             label: "Not sure"
-        custom_select_error: Please select Yes or No
+        custom_select_error: Select if you received the letter from the NHS
```
Comment (injms) [inline config/locales/en.yml:63]: Ta!

**Review by injms — APPROVED** [review]
An excellent pull request.

**Pull request merged**